chart/flux/templates/deployment.yaml

```diff
@@ -131,6 +131,9 @@ spec:
           {{- if .Values.memcached.createClusterIP }}
           - --memcached-service=
           {{- end }}
+          {{- if not .Values.clusterScope }}
+          - --k8s-namespace-whitelist={{ .Release.Namespace }}
```

Comment:
The whitelist flag has been deprecated, please use `--k8s-allow-namespace` https://github.com/weaveworks/flux/blob/master/cmd/fluxd/main.go#L166

Reply:
changed